src/views/SignIn/SignIn.js

```diff
@@ -107,6 +107,7 @@ const SignIn = ({ logInAction, auth }) => {
               <Grid container direction="row" justifyContent="center" alignItems="baseline">
                 <Grid item>
                   <Select
+                    variant="standard"
```

Comment:
same as above, what style has been chosen for inputs, _standard_ or default _outlined_ ? :slightly_smiling_face:

Reply:
Same as above.